dags/main_summary.py

```diff
@@ -43,6 +45,94 @@
     email=["telemetry-alerts@mozilla.com", "relud@mozilla.com", "jklukas@mozilla.com"],
     dag=dag)
 
+gcloud_docker_image = "google/cloud-sdk:263.0.0-slim"
+main_summary_dataproc_bucket = "gs://moz-fx-data-derived-datasets-parquet-tmp"
+main_ping_bigquery_export_prefix = main_summary_dataproc_bucket + "/export"
+main_ping_bigquery_export_dest = main_ping_bigquery_export_prefix + "/submission_date={{ds}}/document_namespace=telemetry/document_type=main/document_version=4/*.avro"  # noqa
+main_ping_bigquery_export = gke_command(
+    task_id="main_ping_bigquery_extract",
+    command=[
+        "bq",
+        "extract",
+        "--destination_format=AVRO",
+        "moz-fx-data-shared-prod.payload_bytes_decoded.telemetry_telemetry__main_v4${{ds_no_dash}}",
+        main_ping_bigquery_export_dest,
+    ],
+    docker_image=gcloud_docker_image,
+)
+
+main_summary_dataproc = SubDagOperator(
+    subdag=moz_dataproc_jar_runner(
+        parent_dag_name="main_summary",
+        dag_name="main_summary_dataproc",
+        default_args=default_args,
+        cluster_name="main-summary-{{ds}}",
+        image_version="1.3",
+        idle_delete_ttl="600",
+        worker_machine_type="n1-standard-8",
+        num_preemptible_workers=40,
+        optional_components=[],
+        install_component_gateway=False,
+        jar_urls=[
+            "s3a://net-mozaws-data-us-west-2-ops-ci-artifacts/mozilla/telemetry-batch-view/master/telemetry-batch-view.jar",
```

Comment:
I'm assuming this is s3 bucket is public, since I don't see any aws credentials configured

Reply:
At some point we'll need to move the CI and location to GCS?

Reply:
> I'm assuming this is s3 bucket is public, since I don't see any aws credentials configured

yes

> At some point we'll need to move the CI and location to GCS?

at some point, yes